static/app/components/smartSearchBar/index.tsx

```diff
@@ -1288,10 +1288,13 @@ class SmartSearchBar extends React.Component<Props, State> {
             ? this.cursorPosition
             : valueToken.location.end.offset;
         } else {
-          // Include everything after the ':'
           const keyLocation = cursorToken.key.location;
           clauseStart = keyLocation.end.offset + 1;
           clauseEnd = location.end.offset + 1;
+          // The user tag often contains : within its value and we need to quote it.
```

Comment:
carried this over from the old search logic, a little unsure if we need it but doesn't hurt for now